components/StarRating.tsx

```diff
@@ -0,0 +1,18 @@
+function StarRating({ rating }: { rating: number }) {
+  return (
+    <div className="flex items-center">
+      {[...Array(5)].map((_, index) => (
+        <span
+          key={index}
+          className={`text-xl ${
+            index < rating ? "text-yellow-400" : "text-gray-300"
```

Comment:
we can use a utility like `classnames` to make this more simpler, and handle more complex composition of classnames